aksel.nav.no/website/components/sanity-modules/code-examples/CodeExamples.tsx

```diff
@@ -161,6 +161,7 @@ const ComponentExamples = ({ node }: CodeExamplesProps) => {
                       invisible: unloaded,
                     },
                   )}
+                  style={{ maxHeight: "calc(100vh - 200px)" }}
```

Comment:
This is to prevent the preview from taking up the entire viewport height.

Reply:
Can do this in className
```
"max-h-[calc(100vh-200px)]",
```